docs/roadmap.rst

```diff
@@ -0,0 +1,223 @@
+The ``tedana`` Roadmap
+======================
+
+Project vision
+--------------
+
+ME-EPI processing is not well integrated into major preprocessing packages,
+yielding duplicated and unmaintained code.
+```tedana`` has been developed to address this need and will serve as a central repository
+for standard ME-EPI denoising as well as a testing ground for novel ME-EPI denoising methods.
+This will jointly reduce the external burden on pipeline maintainers,
+facilitate increased ME-EPI adoption, and enable future development in ME-EPI denoising.
+
+Metrics of success and corresponding milestones
+-----------------------------------------------
+
+We will know that we have been successful in creating ``tedana`` when we have succeeded in providing
+several concrete deliverables, which can be broadly categorized into:
+
+1. *:ref:`Documentation`*,
+2. *:ref:`Transparent and Reproducible Processing`*,
+3. *:ref:`Testing`*,
+4. *:ref:`Workflow Integrations`*,
+5. *:ref:`Extensions and Improvements to ME-EPI processing`*, and
+6. *:ref:`Developing a healthy community`*
+
+Each deliverable has been synthesized into a milestone that gives the ``tedana`` community a link
+between the issues and the high level vision for the project.
+
+.. _Documentation:
+
+Documentation
+`````````````
+**Summary**:
+One long-standing concern with ME-EPI denoising has been the availability of
+documentation for the method outside of published scientific papers.
+To address this, we have created `a ReadTheDocs site`_;
+however, there are still several sections either explicitly marked as "# TODO"
+or otherwise missing crucial information.
+
+We are committed to providing helpful documentation for all users of ``tedana``.
+One metric of success, then, is to develop documentation that includes:
+
+1. Motivations for conducting echo time dependent analysis,
```

Comment:
Feedback from Javier suggest that we should also include guidance on collecting ME data, or at least pointers to appropriate resources.